python/dolfinx/la/__init__.py

```diff
@@ -19,9 +20,9 @@
     "MatrixCSR",
     "Norm",
     "Vector",
-    "create_petsc_vector",
     "is_orthonormal",
     "matrix_csr",
+    "norm",
```

Comment:
Norm used anywhere?

Reply:
Yes, extensively in the unit tests for example.